doc-content/shared-kie-docs/src/main/asciidoc/KIE/Overview/architecture-scenarios-ref.adoc

```diff
@@ -0,0 +1,71 @@
+[id='architecture-scenarios-ref_{context}']
```

Comment:
Not sure we chose best examples here or how they are explained. I think this might need some more discussion.

Reply:
@ron-murhammer, these examples better now? What do you think about the "Scenario advantages" now?

Reply:
Definitely looks better. I do agree with Alex that diagrams would be useful.